pkg/metadata/host/host.go

```diff
@@ -152,12 +152,31 @@ func getHostAliases() []string {
 	return aliases
 }
 
+// try to get the public IPv4 address using AWS and GCE
+func getPublicIPv4() (string, error) {
+	publicIPv4, err := ec2.GetPublicIPv4()
+	if err == nil {
+		log.Infof("EC2 public IP = %s", publicIPv4)
+		return publicIPv4, err
+	}
+
+	publicIPv4, err = gce.GetPublicIPv4()
```

Comment:
Is there an Azure equivalent?

Reply:
Looks like it, but I talked to Lee and were punting on it for now: https://datadoghq.atlassian.net/browse/NET-1038